worker/__init__.py

```diff
@@ -88,11 +89,10 @@ def fetch_events(self, lat, lon, rad, **params):
                 end_time_local=event['end']['local'],
                 timezone=event['start']['timezone'],
                 capacity=event['capacity'],
-                location={
-                    "lat": venue['latitude'],
-                    "lon": venue['longitude'],
-                    "venue_id": event['venue_id']
-                },
+                location=WKTElement('POINT({} {})'.format(
```

Comment:
Why is `WKTElement` used here, but `Geography` is used at `beluga/models.py:85`?

Reply:
Directly creating and inserting `Geography` objects wasn't working for me 🤔. I'm sure theres a way to do it, and it would probably clean up that testing approach.